.eslintignore

```diff
@@ -0,0 +1,9 @@
+dist/
+docs/
+examples/
+node_modules/
+lib/
+scripts/
+# for disable website but should be done as well
```

Comment:
@yungsters haha! yeah that's what I meant, not sure my comment is even in English lol